kubernetes-client/src/main/java/io/fabric8/kubernetes/client/KubernetesClient.java

```diff
@@ -157,6 +159,17 @@
    */
   @Deprecated
   <T extends HasMetadata, L extends KubernetesResourceList<T>> MixedOperation<T, L, Resource<T>> customResources(CustomResourceDefinitionContext crdContext, Class<T> resourceType, Class<L> listClass);
+  
+  /**
+   * Semi-Typed API for managing CustomResources.
+   *
+   * @param crdContext CustomResourceDefinitionContext describes the core fields used to search for CustomResources
+   * @return returns a MixedOperation object with which you can do basic CustomResource operations
+   */
+  default MixedOperation<GenericKubernetesResource, GenericKubernetesResourceList, Resource<GenericKubernetesResource>> genericCustomResources(
```

Comment:
I think this method can also be used for regular Kubernetes resources. It seems to address (https://github.com/fabric8io/kubernetes-client/issues/2929) . Do you think `genericKubernetesResources()` or `dynamic()` would be better?

Reply:
> Do you think genericKubernetesResources() or dynamic() would be better?

I wasn't sure what to name it.  Definitely if its usage with regular resources isn't discouraged, then the names you are suggesting make sense.  Generic is consistent with the resource class names, but is more overloaded as a term.  I'm fine either way - @manusa WDYT?

Reply:
I'll update to genericKubernetesResources